lib/wda/xcodebuild.js

```diff
@@ -52,6 +52,7 @@ class XcodeBuild {
     this.wdaRemotePort = args.wdaRemotePort;
 
     this.updatedWDABundleId = args.updatedWDABundleId;
+    this.derivedDataPath = args.derivedDataPath;
```

Comment:
Why not just `this._derivedDatPath = args.derivedDataPath;`? Then skip lines 94-97 below?

Reply:
To be honest I wasnt sure if that would break something. Going to change that and run a few tests.